packages/js/plugins/http/src/schema.graphql

```diff
@@ -1,33 +1,11 @@
-type Header {
-  key: String!
-  value: String!
-}
-
-type UrlParam {
-  key: String!
-  value: String!
-}
-
-type Response {
-  status: Int!
-  statusText: String!
-  headers: [Header!]
-  body: String
-}
-
-type Request {
-  headers: [Header!]
-  urlParams: [UrlParam!]
-  responseType: ResponseType!
-  body: String
-}
+#import { Module } into Http from "ens/http.polywrap.eth"
 
-enum ResponseType {
-  TEXT
-  BINARY
+type Env {
+  """
+  Disable querying providers in parallel when resolving URIs
+  """
+  disableParallelRequests: Boolean
```

Comment:
This doesn't seem to be used anywhere, can we remove it? Not sure it makes sense for this wrapper.